data/parse_smogon_stats.py

```diff
@@ -41,64 +38,29 @@ def get_smogon_stats_file_name(game_mode, month_delta=1):
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        item = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if item != OTHER_STRING:
-                            pokemon_information[pokemon_name][ITEM_STRING].append((item, percentage))
-
-            elif normalize_name(segment) == MOVES_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        move = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if move != OTHER_STRING:
-                            if constants.HIDDEN_POWER in move:
-                                move = "{}{}".format(move, constants.HIDDEN_POWER_ACTIVE_MOVE_BASE_DAMAGE_STRING)
-                            pokemon_information[pokemon_name][MOVES_STRING].append((move, percentage))
-
-            elif normalize_name(segment) == ABILITY_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        ability = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if ability != OTHER_STRING:
-                            pokemon_information[pokemon_name][ABILITY_STRING].append((ability, percentage))
-
-    return pokemon_information
+    infos = r.json()['data']
+    final_infos = {}
+    for x in infos.keys():
+        spreads = []
+        items = []
+        moves = []
+        abilities = []
+        final_infos[normalize_name(x)] = {}
+        for t in infos[x]['Spreads']:
+            if float("{:.16f}".format(float(infos[x]['Spreads'][t]))) != 0:
+                spreads.append((normalize_name(t.split(':')[0]), normalize_name(t.split(':')[1].replace('/', ',')), float("{:.16f}".format(float(infos[x]['Spreads'][t])))))
+        for j in infos[x]['Items']:
+            if infos[x]['Items'][j] != 0:
+                items.append((j, infos[x]['Items'][j]))
+        for k in infos[x]['Moves']:
+            if infos[x]['Moves'][k] != 0:
+                moves.append((k, infos[x]['Moves'][k]))
+        for l in infos[x]['Abilities']:
+            if infos[x]['Abilities'][l] != 0:
+                abilities.append((l, infos[x]['Abilities'][l]))
+        final_infos[normalize_name(x)][SPREADS_STRING] = spreads
+        final_infos[normalize_name(x)][ITEM_STRING] = items
+        final_infos[normalize_name(x)][MOVES_STRING] = moves
+        final_infos[normalize_name(x)][ABILITY_STRING] = abilities
```

Comment:
Another optimization that I'll do (outside of this PR) is combine similar spreads together, since it makes no sense to classify these as different spreads:

```python
('timid', '0,0,0,252,4,252', 727.0)
('timid', '0,0,4,252,0,252', 216.0)
```